middlewares/stripPrefixRegex_test.go

```diff
@@ -73,18 +73,16 @@ func TestStripPrefixRegex(t *testing.T) {
 				actualPath = r.URL.Path
 				actualHeader = r.Header.Get(ForwardedPrefixHeader)
 			})
-
 			handler := NewStripPrefixRegex(handlerPath, testPrefixRegex)
-			server := httptest.NewServer(handler)
-			defer server.Close()
 
-			resp, err := http.Get(server.URL + test.path)
-			require.NoError(t, err, "%s: unexpected error.", test.path)
+			req := testhelpers.MustNewRequest(http.MethodGet, "http://localhost"+test.path, nil)
+			resp := &httptest.ResponseRecorder{Code: http.StatusOK}
+
+			handler.ServeHTTP(resp, req)
 
-			assert.Equal(t, test.expectedStatusCode, resp.StatusCode, "%s: unexpected status code.", test.path)
+			assert.Equal(t, test.expectedStatusCode, resp.Code, "%s: unexpected status code.", test.path)
```

Comment:
You don't need to include `test.path` in the error description; the sub-test (which we parameterized with the path) will do that for you.